File: app/src/main/java/be/appfoundry/aipdemo/database/AppDatabase.java
package be.appfoundry.aipdemo.database;

import com.raizlabs.android.dbflow.annotation.Database;

@Database(name = AppDatabase.DATABASE_NAME, version = AppDatabase.VERSION)
public class AppDatabase {

    public static final String DATABASE_NAME = "AIPDemoDatabase";
    public static final int VERSION = 1;

}

File: app/src/main/java/be/appfoundry/aipdemo/data/database/AppDatabase.kt
package be.appfoundry.aipdemo.data.database

import androidx.room.Database
import androidx.room.RoomDatabase
import be.appfoundry.aipdemo.data.model.Card

private const val DATABASE_VERSION = 1

@Database(entities = [Card::class], version = DATABASE_VERSION)
abstract class AppDatabase : RoomDatabase() {
    abstract fun cardDao(): CardDao

    companion object {
        const val DATABASE_NAME = "app-database"
    }
}